packages/common/types/control.types.ts

```diff
@@ -37,6 +37,8 @@ export interface ControlBase {
   placeholder?: string;
   validators?: string[]; // TODO: implement validator type
   options?: string[] | ControlOption[];
+  rows?: number;
+  cols?: number;
```

Comment:
I think we should remove these props from the ControlBase, it should contain only the common props.